packages/main/src/Table.js

```diff
@@ -274,7 +274,7 @@ const metadata = {
 		},
 
 		/**
-		 * Fired when the <code>ui5-table-column</code> is shown as a pop-in instead of hiding it.
+		 * Fired when the component is shown as a pop-in instead of hiding it.
```

Comment:
Here we  mention the `<code>ui5-table-column</code>`, perhaps we should leave it or replace it with `component column`